Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -820,6 +827,11 @@ void dropDownContent(ImGui gui) {
                     }
                 }
 
+                if (type <= ProductDropdownType.Ingredient && spentFuelRecipes.Length > 0) {
+                    gui.BuildInlineObjectListAndButton(spentFuelRecipes, DataUtils.AlreadySortedRecipe, (x) => { spentFuel = goods; addRecipe(x); }, "Produce it as a spent fuel", type >= ProductDropdownType.Product ? 6 : 3, true, recipeExists);
```

Comment:
I ask you to please keep the lines shorter than 120 characters, so they fit into most monitors. This one is 245 :D
As one of the things, you can define the `Action` and the tertiary separately.

Reply:
In my defense, it's a direct copy of the 4 other calls to `BuildInlineObjectListAndButton` right below.

I'll see if i can trim down the methods parameter list and/or clean up the way it's called with some newlines or variable.

Reply:
> it's a direct copy of the 4 other calls

I understand it. 
However, when some of my takes may be debatable, I doubt that the argument "It's bad when the code requires horizontal scrolling" can be outweighed by nearly anything else.

If the whole file is like that, then I'll have to reformat the whole file eventually.

Reply:
Done

Reply:
> > it's a direct copy of the 4 other calls
> 
> I understand it. However, when some of my takes may be debatable, I doubt that the argument "It's bad when the code requires horizontal scrolling" can be outweighed by nearly anything else.
> 
> If the whole file is like that, then I'll have to reformat the whole file eventually.

Nah i agree, it's a valid comment. 120 chars is a very tight budget especially on some of the UI code, i would personally try to avoid splitting up the text strings over multiple lines for example. But methods like this one, you can easily put each argument on a new line, or at least when it approaches 120 chars. By doing so, i even noticed one of the inline ternaries was always true, so i replaced them with constants.